README.md

```diff
@@ -87,19 +24,20 @@ This can be enabled by setting `enforce_trusted_endpoints` in your Grafana confi
 enforce_trusted_endpoints = true
 ```
 
-## Configuring the datasource in Grafana
-
-The steps for configuring the integration between the Azure Data Explorer service and Grafana are:
+## Configure the Azure Data Explorer data source
 
+To configure ADX for using this data source:
 1. Create an Azure Active Directory (AAD) Application and AAD Service Principle.
-2. Log into the Azure Data Explorer WebExplorer and connect the AAD Application to an Azure Data Explorer database user.
-3. Use the AAD Application to configure the datasource connection in Grafana.
+1. In the Azure Data Explorer WebExplorer and connect the AAD Application to an Azure Data Explorer database user.
+1. Use the AAD Application to configure the data source connection in Grafana.
+
+1. (Optional) To use the dropdown cluster select when creating queries, add reader access for Azure Resource Graph queries, and add reader access to each desired cluster.
```

Comment:
I'm fairly certain `Reader` access isn't needed for each cluster in addition to each subscription? It's either or right?

Reply:
We have reader access to the grafana cluster we do not own and cannot see it in the cluster select.

Reply:
In that case we _should_ be able to remove the `and add reader access to each desired cluster.` line as it's the subscription reader access that's relevant